en/flight_modes/mission.md

```diff
@@ -29,8 +29,11 @@ At high level all vehicle types behave in the same way when MISSION mode is enga
 
 1. If a mission is stored and PX4 is flying it will execute the [mission/flight plan](../flying/missions.md) from the current step. 
 1. If a mission is stored and PX4 is landed:
-   * On copters PX4 will execute the [mission/flight plan](../flying/missions.md). If the mission does not have a `TAKEOFF` command then PX4 will fly the vehicle to the minimum altitude before executing the remainder of the flight plan from the current step.
-   * On fixed-wing vehicles PX4 will not automatically take off (the autopilot will detect the lack of movement and set the throttle to zero). The vehicle may start executing the mission if hand- or catapult- launched while in mission mode.  
+
+   - On copters PX4 will execute the [mission/flight plan](../flying/missions.md).
+     If the mission does not have a `TAKEOFF` command then PX4 will fly the vehicle to the minimum altitude before executing the remainder of the flight plan from the current step.
+   - On fixed-wing vehicles PX4 will not automatically take off (the autopilot will detect the lack of movement and set the throttle to zero).
+     If the currently active waypoint is a Takeoff, the system will automatically takeoff (see [FW Takeoff/Landing in Mission](#fixed-wing-mission-takeoff-landing)).

```

Comment:
@sfuhrer @tstastny Just to be very clear here:
- In takeoff mode a catapult launch is triggered by acceleration detection - either throwing or shaking to start the throttle ramping up.
  - What this implies is that in a mission mode you do not need acceleration detection for catapult launch - is that correct? That does not make sense to me - I'd expect the vehicle to arm, but would still expect it to require throwing.
  - However for runway mode I'd expect it to just take off. Can you still "nudge" as in takeoff mode?
- I assume all the parameter setup for catapult/runway are required whether using mission or takeoff?
- If I do a catapult launch in mission I am throwing at a heading that might not match the takeoff item heading. Does the vehicle just track to that takeoff mission item heading as soon as it can?

Reply:
Actually the launch detection is only an additional feature, that's not necessary for hand-launch. If it's not enabled the motor will just immediately ramp up on arming. 

>What this implies is that in a mission mode you do not need acceleration detection for catapult launch - is that correct? That does not make sense to me - I'd expect the vehicle to arm, but would still expect it to require throwing.

Mission mode doesn't change the procedure - you arm the vehicle, and if launch detection is enabled it needs to detect an acceleration before it starts the motor. 

>However for runway mode I'd expect it to just take off. Can you still "nudge" as in takeoff mode?

You can only nudge if in runway takeoff mode.

>I assume all the parameter setup for catapult/runway are required whether using mission or takeoff?

Correct, there is no difference in required params.